src/scss/components/carousel.scss

```diff
@@ -2,7 +2,7 @@
   // The navbar has a 1px border on the bottom, so adding it to the margin
   margin-top: -($line-height-computed + 1);
```

Comment:
You could probably set this this to just '-$line-height-computed' which comes out as around 2.4rem (24px).
It actually is needed to compensate for the nav margin which is currently set to 20px in starter.